src/config.ts

```diff
@@ -29,12 +29,20 @@ export class Config {
   static async getConfig(octokit: CustomOctokit): Promise<Config> {
     const path = getInput('config-path', { required: true });
 
+    const overwriteMerge = (
+      _defaultArray: string[],
+      configArray: string[],
+      _options?: deepmerge.Options
+    ) => configArray;
+
     const retrievedConfig = (
       await octokit.config.get({
         ...context.repo,
         path,
         defaults: configs =>
-          deepmerge.all([this.defaults, ...configs]) as Partial<Config>,
+          deepmerge.all([this.defaults, ...configs], {
+            arrayMerge: overwriteMerge,
```

Comment:
`arrayMerge` is not specified in config.ts, or is it part of TypeScript programming language?

Reply:
Now I have fount a definition.

Reply:
It's a configuration option for the deepmerge library.